test/e2e-tests-auction.sh

```diff
@@ -6,7 +6,7 @@ cd dex-contracts/
 EXPECTED_AUCTION=0
 
 # Ensure no orders yet in auction slot 1
-mongo dfusion2 --eval "db.orders.find({'auctionId': ${EXPECTED_AUCTION}}).size()" | grep 0
+mongo dfusion2 --eval "db.orders.find({'auctionId': ${EXPECTED_AUCTION}}).size()" | grep -w 0
```

Comment:
What does the -w mean?

Reply:
Match whole word. You are very likely to find a 0 somewhere in the sessionID (and thus this command used to pass even if the auction size was not 0)